apps/communications/contacts/js/views/details.js

```diff
@@ -435,68 +425,18 @@ contacts.Details = (function() {
   };
 
   var renderSocial = function cd_renderSocial(contact) {
```

Comment:
Yes, we need it since the share button is render in this method. However I will suggest to rename it to renderShareButton, but I think it's better to modify it in bug1181112